change/react-native-windows-3e2778f7-39fc-4d9a-9fc9-3b8c68a0975f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "[0.63] codegen test fix for build on new agents",
```

Comment:
nit: These show up in release notes, so we usually avoid prefixing with a version. `yarn change` will let you type in anything for these, so it's sometimes worth giving a more "user-facing" description than what would normally be in a commit message.

Reply:
This is an example of what these look like, if you go to the "GitHub" releases view, or receive notifications from the repo. 
![image](https://user-images.githubusercontent.com/835219/111719869-ff141c00-8819-11eb-9670-d03edfe53552.png)